contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -299,6 +299,11 @@ def risk(self) -> list[dict[str, Any]]:
                 risk_object['threat_object_field'] = entity.name
                 risk_object['threat_object_type'] = "url"
                 risk_objects.append(risk_object)
+            
+            elif 'Attacker' in entity.role:
```

Comment:
Note that I tested an ESCU build made using this branch against the new testing in my branch (see #241). My risk/observable matching showed 50+ detections generating risk events for Attacker observables, which shouldn't happen. These changes in your PR Lou resolved all 50+ of those detections.